### DISCUSSION TITLE
One NFT but iBax NFT☠️

### DISCUSSION THREAD

**Discussion by Azoahai** [IBAX-io/go-ibax#544] [Q&A] [answered]
![IMG_20220124_165940](https://user-images.githubusercontent.com/95991723/150798629-cbda2307-40d3-468a-849b-7dbe8aa7461e.jpg)

**Comment by MstfYtskn** [comment]
Bu ne?

**Comment by ibaxio** [comment, marked as answer]
This is nothing to do with IBAX Network.
